swagger/function-manager.yaml

```diff
@@ -421,6 +421,10 @@ definitions:
         type: array
         items:
           type: string
+      httpContext:
```

Comment:
seems like this attribute should be `readOnly`, as it cannot be manually set when executing a function through API.

Reply:
Yeah it should probably be `readOnly`. I'll make the change.